app/projects/models.py

```diff
@@ -116,7 +116,7 @@ def __str__(self):
 class Participant(models.Model):
     user = models.OneToOneField(User)
     data_challenge = models.ForeignKey(DataProject)
-    team = models.ForeignKey(Team, null=True, blank=True)
+    team = models.ForeignKey(Team, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
Does this mean the user's forms would get deleted too? If they are members of a team..

Reply:
This specifically did not delete the forms. Forms are connected to users, not participants. Maybe we should change that sometime..